src/application-status-overlay/ApplicationStatusOverlayProvider.jsx

```diff
@@ -69,6 +69,15 @@ const ApplicationStatusOverlayProvider = ({ children, scrollRefCallback, ...cust
   const registeredVariant = registeredStatusOverlay[lastRegisteredStatusOverlayKey]?.variant;
 
   useEffect(() => {
+    const { head } = document;
+    if (!head.querySelector('link#inert-style')) {
+      if (head.querySelector('style#inert-style')) {
+        head.removeChild(head.querySelector('style#inert-style'));
+      }
```

Comment:
If the style block has already been added, the csp violation was caused by some other inert poly fill and we may as well leave it there, i think.

Reply:
Updated here https://github.com/cerner/terra-application/pull/89/commits/30513765b7611c3c5f527f56573286f521ce9c33.